.github/workflows/lol-hackathoon.yml

```
@@ -0,0 +1,310 @@
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Kubectl
+        uses: azure/setup-kubectl@v1
+        with:
+          version: 'v1.20.0'
+
+      - name: Configure Kubeconfig
+        env:
+          KUBECONFIG_STAGING: ${{ secrets.KUBECONFIG_STAGING }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2317:info:6:1: Command appears to be unreachable. Check usage (or ignore if invoked indirectly)

(shellcheck)